app/scripts/views/complete_sign_up.js

```diff
@@ -38,7 +37,15 @@ define(function (require, exports, module) {
       this._verificationInfo = new VerificationInfo(searchParams);
       var uid = this._verificationInfo.get('uid');
 
-      this._account = options.account || this.user.getAccountByUid(uid);
+      var account = options.account || this.user.getAccountByUid(uid);
+      // the account will not exist if verifying in a second browser, and the
+      // default account will be returned. Add the uid to the account so
+      // verification can still occur.
+      if (account.isDefault()) {
+        account.set('uid', uid);
```

Comment:
Apologies if I missed it, but I couldn't see any assertions about this call in `app/tests/spec/views/complete_sign_up.js`. Should it have some?